recipes-graphics/wayland/weston_4.0.0.imx.bb

```diff
@@ -19,8 +19,10 @@ S = "${WORKDIR}/git"
 UPSTREAM_CHECK_URI = "https://wayland.freedesktop.org/releases.html"
 
 inherit autotools pkgconfig useradd distro_features_check
-# depends on virtual/egl
-REQUIRED_DISTRO_FEATURES = "opengl"
+# Disable OpenGL for parts with GPU support for 2D but not 3D
+REQUIRED_DISTRO_FEATURES          = "opengl"
+REQUIRED_DISTRO_FEATURES_imxgpu2d = ""
+REQUIRED_DISTRO_FEATURES_imxgpu3d = "opengl"
```

Comment:
Is this working for imx7 which doesn't have any GPU?
I'm asking because previous patch adds Weston for all packages:
https://github.com/Freescale/meta-freescale/pull/9/commits/52364a8346dc008489442b750020ba43da3774c8

Reply:
Yes. Using software rendering I think.

Reply:
Who provides opengl then? Is it using the mesa swrast?